Clients/GUI/ViewModels/SignUpViewModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace GUI.GUIModels
+{
+    public class SignUpViewModel
+    {
+        [Required(ErrorMessage = "Email is required")]
```

Comment:
Can you please explain what are thses attributes for?

Reply:
It's for validation in gui form, at least now we don't have fluent validation yet, mb then we will be able to replace this with fluent validation stuff

Reply:
UPD: yes, it will be done via fluent validation when the validation task will be done